runtime/interpreter/interpreter_expression.go

```diff
@@ -951,8 +956,11 @@ func (interpreter *Interpreter) VisitForceExpression(expression *ast.ForceExpres
 func (interpreter *Interpreter) VisitPathExpression(expression *ast.PathExpression) ast.Repr {
 	domain := common.PathDomainFromIdentifier(expression.Domain.Identifier)
 
-	return PathValue{
-		Domain:     domain,
-		Identifier: expression.Identifier.Identifier,
-	}
+	return NewPathValue(
+		interpreter,
+		common.NewPathMemoryUsage(expression.Identifier.Identifier),
+		func() (common.PathDomain, string) {
+			return domain, expression.Identifier.Identifier
```

Comment:
Good point!

Looking into this, I notice another problem: the decoder is metering the Identifier string but importValue isn't.

I'm going to meter the Identifier separately and meter Path as a constant because then it'll just be a container.